src/main/java/org/opensrp/service/SearchService.java

```diff
@@ -21,8 +21,9 @@ public SearchService(SearchRepository search) {
 	}
 
 
-	@PreAuthorize("hasRole('CLIENT_VIEW')")
-	@PostFilter("hasPermission(filterObject, 'CLIENT_VIEW')")
+	/*
+	No ACL to allow for global searching
+	 */
```

Comment:
This assumes that search will never have ACL. Can we add global methods like on other services and the default we leave as having permissions